dashboard/views/_general/_committee_list.py

```diff
@@ -3,26 +3,30 @@
 from django.urls import reverse, reverse_lazy
 
 from dashboard.forms import CommitteeCreateForm
-from dashboard.models import Brother, Committee
+from dashboard.models import Brother, Committee, query_positions_with_committee, Position
```

Comment:
huh i broke something on master didnt I...  must have forgotten to stick these into the import